packages/docs/docs/advanced/custom-font.mdx

```diff
@@ -0,0 +1,71 @@
+# Custom font
+
+To use a font from hosters like Google Fonts. First make a css file under `src`
+
+```
+root
+└─src
+  ├─scenes/
+  ├─motion-canvas.d.ts
+  ├─project.meta
+  ├─project.ts
+  └─global.css *
+```
+
+Inside `global.css`, import the font using `@import url(your link)`.
+
+```css
+@import url('https://fonts.googleapis.com/css2?family=Fira+Code:wght@400;700&display=swap');
+```
+
+Then, in `project.ts`, import the css file.
+
+```ts
+import {makeProject} from '@motion-canvas/core/lib';
+
+import example from './scenes/example?scene';
+
+import "./global.css" <- import the css
```

Comment:
This should be a comment to make this a valid snippet, in case people will copy it.
```suggestion
import "./global.css"; // <- import the css
```